content/02-types.md

```diff
@@ -596,7 +597,7 @@ This example program calls `test` four times and prints its return value.
 
 The output shows that optional arguments which are omitted from the call have a value of `null`. This implies that the type of these arguments must admit `null` as value, which raises the question of its [nullability](types-nullability). The Haxe Compiler ensures that optional basic type arguments are nullable by inferring their type as `Null<T>` when compiling to a [static target](define-static-target).
 
-While the first three calls are intuitive, the fourth one might come as a surprise: It is indeed allowed to skip optional arguments if the supplied value is assignable to a later argument.
+While the first three calls are intuitive, the fourth one might come as a surprise; optional arguments are allowed to be skipped if the supplied value is assignable to a later argument.
```

Comment:
`… are allowed to be skipped …` sounds a bit like a forced use of the passive.

Reply:
"optional arguments can be skipped" might sound better, but I don't immediately think something's too out of place here. If you have a better recommendation for this please share.